api/components/Examples/Agriculture_services/on_search/on_search_assaying.json

```diff
@@ -0,0 +1,765 @@
+                  "value": "13"
+                },
+                {
+                  "descriptor": {
+                    "code": "val"
+                  },
+                  "value": "{\"type\": \"FeatureCollection\", \"features\": [{\"type\": \"Feature\", \"properties\": {}, \"geometry\": {\"coordinates\": [[[77.175571, 28.675927], [77.128737, 28.600972], [77.446934, 28.545325], [77.175571, 28.675927]]], \"type\": \"Polygon\"}}]}"
+                },
+                {
+                  "descriptor": {
+                    "code": "unit"
+                  },
+                  "value": "geojson"
+                }
+              ]
+            }
+          ],
+          "items": [
+            {
+              "id": "I1",
+              "descriptor": {
+                "name": "Quality testing for Arhar Whole",
+                "code": "UPC code",
+                "short_desc": "Assaying service for Arhar Whole",
+                "long_desc": "Assaying service for Arhar Whole",
+                "images": [
+                  {
+                    "url": "https://abc.com/images/207.png"
+                  }
+                ],
+                "media": [
+                  {
+                    "mimetype": "video/mp4",
+                    "url": "video_url"
+                  }
+                ]
+              },
+              "creator": {
+                "descriptor": {
+                  "name": "nobroker",
+                  "contact": {
+                    "name": "Raj Kumar",
+                    "address": {
+                      "full": "Sector-1, Noida,Uttar Pradesh-201301"
+                    },
+                    "phone": "18001801018",
+                    "email": "consumer.services@nobroker.com"
+                  }
+                }
+              },
+              "price": {
+                "currency": "INR",
+                "value": "1549.00",
+                "offered_value": "1199.00",
+                "maximum_value": "1549.00"
+              },
+              "category_ids": ["AGR12-xxxx"],
+              "fulfillment_ids": ["1"],
+              "location_ids": ["L1"],
+              "payment_ids": ["1"],
+              "cancellation_terms": [
+                {
+                  "fulfillment_state": {
+                    "descriptor": {
+                      "code": "PENDING"
+                    }
+                  },
+                  "cancellation_fee": {
+                    "percentage": "0.00"
+                  }
+                },
+                {
+                  "fulfillment_state": {
+                    "descriptor": {
+                      "code": "IN_TRANSIT"
+                    }
+                  },
+                  "cancellation_fee": {
+                    "percentage": "10.00"
+                  }
+                }
+              ],
+              "tags": [
+                {
+                  "descriptor": {
+                    "code": "TEST_INFORMATION"
```

Comment:
Probably, we need to define tag groups and see , if we really need tags, if Yes, then lets standardize it.